templates/citizen_action/attend.html.twig

```diff
@@ -1,5 +1,7 @@
 {% extends 'events/attend.html.twig' %}
 
+{% block canonical_url url('app_citizen_action_attend', {slug: citizen_action.slug}) %}
```

Comment:
Just a micro CS, can you add whitespaces inside arrays ? `{ slug: citizen_action.slug }`
Same for other